llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -1674,6 +1676,11 @@ class TargetTransformInfo {
 
   /// @}
 
+  std::optional<KnownBits>
```

Comment:
Not much point in making this optional. A direct KnownBits with unknown everything naturally composes without requiring an explicit valid check